packages/frontend-2/pages/projects/[id]/webhooks/index.vue

```diff
@@ -0,0 +1,269 @@
+<template>
+  <div>
+    <Portal to="navigation">
+      <HeaderNavLink :to="projectRoute(projectId)" :name="projectName"></HeaderNavLink>
+      <HeaderNavLink
+        :to="`${projectRoute(projectId)}/webhooks`"
+        name="Webhooks"
+      ></HeaderNavLink>
+    </Portal>
+    <div class="flex items-center justify-between">
+      <h1 class="text-2xl font-bold">Webhooks</h1>
+      <div class="flex gap-2">
+        <FormButton color="secondary" :icon-left="BookOpenIcon" @click="openDocs">
+          Open Docs
+        </FormButton>
+        <FormButton :icon-left="PlusIcon" @click="showNewWebhookDialog = true">
+          Create Webhook
+        </FormButton>
+      </div>
+    </div>
+    <div class="my-8 text-sm">
+      Webhooks allow you to subscribe to a stream's events and get notified of them in
+      real time. You can then use this to trigger ci apps, automation workflows, and
+      more.
+    </div>
+
+    <LayoutTable
+      :columns="[
+        { id: 'enabled', header: 'State', classes: 'col-span-1' },
+        { id: 'data', header: 'Data', classes: 'col-span-5' },
+        {
+          id: 'triggers',
+          header: 'Trigger Events',
+          classes: 'col-span-6 whitespace-break-spaces text-xs'
+        }
+      ]"
+      :items="webhooks"
+      :buttons="[
+        {
+          icon: PencilIcon,
+          label: 'Edit',
+          action: openEditWebhookDialog,
+          class: 'text-primary'
+        },
+        {
+          icon: TrashIcon,
+          label: 'Delete',
+          action: openDeleteWebhookDialog,
+          class: 'text-red-500'
+        }
+      ]"
+    >
+      <template #enabled="{ item }">
+        <FormSwitch
+          :model-value="(item.enabled as boolean)"
```

Comment:
It's not a boolean tho, it's a `boolean | null | undefined`. You don't need to do type casting here (which should be done as rarely as possible), you can just convert it to a bool w/ `!!item.enabled`